traits-stubs/traits-stubs/api.pyi

```diff
@@ -19,8 +19,27 @@ from .traits import (
 )
 
 from .has_traits import (
+    ABCHasStrictTraits as ABCHasStrictTraits,
```

Comment:
No, these aren't alphabetized; for ease of comparison, I'm using the same order as is currently in `api.py`.